docker/Dockerfile

```diff
@@ -7,6 +7,9 @@ ENV LANG=C.UTF-8 LANGUAGE=C.UTF-8 LC_ALL=C.UTF-8
 # depends on this library and it wasn't available in the base container
 
 RUN apt-get update --fix-missing \
+    && apt-get upgrade -y \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
```

Comment:
hmm, just curios, does this package need to be installed separately from the others?

Reply:
but it's the same in the main repo, so all good, just my curiosity kicking in 😄

Reply:
Yeah this needed to be installed before installing the certificates-ca package that depends on it but cannot resolve it for some reason.